pythonAPIClient/client.py

```diff
@@ -391,3 +391,40 @@ def search_camera(self, latitude=None, longitude=None, radius=None, camera_type=
         for current_object in camera_response_array:
             camera_processed.append(Camera.process_json(**current_object))
         return camera_processed
+
+    def get_change_log(self, start=None, end=None, offset=None):
+        """
+        Parameters
+        ----------
+        start : str, optional
+            Start time of the log user desires to query
+        end : str, optional
+            End time of the log user desires to query
+        offset : str, optional
+            How many logs to skip
+
+        Returns
+        -------
+            A list of objects containing cameraID and creation time of the log.
+
+        """
+        url = Client.base_URL + 'apps/db-change'
+        if self.token is None:
+            self.request_token()
+        header = self.header_builder()
+        param = {'start': start,
+                 'end': end,
+                 'offset': offset}
+        response = self._check_token(response=requests.get(url, headers=header, params=param),
+                                     flag='GET', url=url, params=param)
+        if response.status_code != 200:
+            if response.status_code == 401:
+                raise AuthenticationError(response.json()['message'])
+            elif response.status_code == 404:
+                raise ResourceNotFoundError(response.json()['message'])
```

Comment:
we should remove this 404 error checking here. this route does not thrown 404 error. however, call this python function may return a 404 error inside request token function. if 404 error is thrown there, it will never reach this line